doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -75,11 +75,38 @@ Note: It will only target the functionality that are exposed by standard OpenSta
 
 #### 4.3.4.1 Identity - Keystone
 
-It covers the test cases against identity management operations like user management, project management, multi-tenancy etc.
-
-| Test case # | sub-category | Description | Requirement # |
-|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
-| `cert.test.vim.01` | API | Show API endpoint catalog. It *must* have endpoint for each Core service. | `req.int.api.01` |
+Keystone API is covered in the OpenStack Gates via
+[Tempest](https://opendev.org/openstack/tempest) and
+[keystone-tempest-plugin](https://opendev.org/openstack/keystone-tempest-plugin)
+as integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml).
+
+According to
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
```

Comment:
Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`